tools/spinel-cli/sniffer.py

```
@@ -0,0 +1,153 @@
+
+   Save stream to file or pipe:
+       ./sniffer.py > trace.pcap
+"""
+
+import sys
+import optparse
+
+import spinel.util as util
+import spinel.config as CONFIG
+from spinel.const import SPINEL
+from spinel.codec import WpanApi
+from spinel.stream import StreamOpen
+from spinel.pcap import PcapCodec
+
+
+# Nodeid is required to execute ot-ncp for its sim radio socket port.
+# This is maximum that works for MacOS.
+DEFAULT_NODEID = 34    # same as WELLKNOWN_NODE_ID
+DEFAULT_CHANNEL = 11
+
+def parse_args():
+    """ Parse command line arguments for this applications. """
+
+    args = sys.argv[1:]
+
+    opt_parser = optparse.OptionParser()
+    opt_parser.add_option("-u", "--uart", action="store",
+                          dest="uart", type="string")
+    opt_parser.add_option("-p", "--pipe", action="store",
+                          dest="pipe", type="string")
+    opt_parser.add_option("-s", "--socket", action="store",
+                          dest="socket", type="string")
+    opt_parser.add_option("-n", "--nodeid", action="store",
+                          dest="nodeid", type="string", default=str(DEFAULT_NODEID))
+
+    opt_parser.add_option("-q", "--quiet", action="store_true", dest="quiet")
+    opt_parser.add_option("-v", "--verbose", action="store_false", dest="verbose")
+    opt_parser.add_option("-d", "--debug", action="store",
+                          dest="debug", type="int", default=CONFIG.DEBUG_ENABLE)
+    opt_parser.add_option("-x", "--hex", action="store_true", dest="hex")
+
+    opt_parser.add_option("-c", "--channel", action="store",
+                          dest="channel", type="int", default=DEFAULT_CHANNEL)
+
+    return opt_parser.parse_args(args)
+
+def sniffer_init(wpan_api, options):
+    """" Send spinel commands to initialize sniffer node. """
+    wpan_api.queue_register(SPINEL.HEADER_DEFAULT)
+    wpan_api.queue_register(SPINEL.HEADER_ASYNC)
+
+    wpan_api.cmd_send(SPINEL.CMD_RESET)
```

You should set `PHY_ENABLED` and just ignore any errors it returns.